### PR TITLE
Esdts index

### PR DESCRIPTION
**Pull request by miiu96**
- Updated Elasticsearch version from the docker-compose configuration.
- Added mappings for `esdts` index.
- Index all the tokens in the `esdts` index.

### CODE REVIEW

File: process/dataindexer/constants.go [hunk 1]
```diff
@@ -43,6 +43,8 @@ const (
 	DelegatorsIndex = "delegators"
 	// OperationsIndex is the Elasticsearch index for transactions and smart contract results
 	OperationsIndex = "operations"
+	// ESDTsIndex is the Elasticsearch index for esdts tokens
```
Comment (bogdan-rosianu) [inline process/dataindexer/constants.go:46]: esdt tokens*

File: process/elasticproc/elasticProcessor.go [hunk 2]
```diff
@@ -499,11 +500,17 @@ func (ei *elasticProcessor) SaveTransactions(
 }
 
 func (ei *elasticProcessor) prepareAndIndexRolesData(tokenRolesAndProperties *tokeninfo.TokenRolesAndProperties, buffSlice *data.BufferSlice) error {
```
Comment (iulianpascalau) [inline process/elasticproc/elasticProcessor.go:502]: please refactor this:
```GO
func (ei *elasticProcessor) prepareAndIndexRolesData(tokenRolesAndProperties *tokeninfo.TokenRolesAndProperties, buffSlice *data.BufferSlice) error {
	err := ei.indexToken(tokenRolesAndProperties, buffSlice, elasticIndexer.TokensIndex)
	if err != nil{
		return err
	}
	
	return ei.indexToken(tokenRolesAndProperties, buffSlice, elasticIndexer.ESDTsIndex)
}

func (ei *elasticProcessor) indexToken(tokenRolesAndProperties *tokeninfo.TokenRolesAndProperties, buffSlice *data.BufferSlice, index string) error{
	if !ei.isIndexEnabled(index) {
		return nil
	}
	
	
	return ei.logsAndEventsProc.SerializeRolesData(tokenRolesAndProperties, buffSlice, index)
}
```

File: process/elasticproc/updateTokenType.go [hunk 3]
```diff
@@ -11,6 +11,13 @@ import (
 )
 
 func (ei *elasticProcessor) indexTokens(tokensData []*data.TokenInfo, updateNFTData []*data.NFTDataUpdate, buffSlice *data.BufferSlice) error {
+	if ei.isIndexEnabled(elasticIndexer.ESDTsIndex) {
```
Comment (iulianpascalau) [inline process/elasticproc/updateTokenType.go:14]: please extract common code & refactor as suggested in previous comment

**Review by iulianpascalau — CHANGES_REQUESTED** [review]

File: process/dataindexer/constants.go (hunk 1 above)
Comment (miiu96) [reply process/dataindexer/constants.go:46]: done

File: process/elasticproc/updateTokenType.go (hunk 3 above)
Comment (miiu96) [reply process/elasticproc/updateTokenType.go:14]: refactored.

**Review by iulianpascalau — DISMISSED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**